Lombiq.HelpfulLibraries.Samples/Lombiq.HelpfulLibraries.Samples.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+    <IsPackable>false</IsPackable>

```

Comment:
I didn't add this to the sln, but this will prevent it publishing if it is ever added.